Software/vortex_ws/src/motion_planning/README.md

```diff
@@ -1 +1,192 @@
-Motion planning package
\ No newline at end of file
+# motion planning and control module package
+
+## description
+
+this package represents four modules (localization,global planner, local planner and motion control )
+
+Localization       : determine AUV's place in the 3d space around it and also determines the location of the obstacles and objects surrounding it.
+
+The Global Planner : give the AUV the ability to plan the correct and feasible path through the starting point to its goal.
+The Local Planner  :generate a set of control commands that‏ moves the AUV on the path generated by the Global Planner but with taking the dynamics into consideration while also trying to avoid obstacles.
+
+motion control :translate the set of control commands from the Local Planner into actions for the thrusters‏
+
+### License
+
+The source code is released under a [GNU GENERAL PUBLIC LICENSE].
+
+**Author: vortex-co<br />
+Affiliation: [VorteX-Co](https://vortex-co.com/home)<br />
+Maintainer: vortex-co, info@vortex-co.com**
+
+The Vision module package has been tested under [ROS2] eloquent on Ubuntu 18.04 
+
+## Table of contents
+* [Installation](#Installation)
+* [Usage](#Usage)
+* [Config files](#Config-files)
+* [Launch files](#Launch-files)
+* [Hardware](#hardware)
+
+
+## Installation
+### Installation from Packages
+
+To install all packages from the this repository as Debian packages use
+
+    sudo apt-get install ros-noetic-...
+    
+Or better, use `rosdep`:
+
+	sudo rosdep install --from-paths src
+
+### Building from Source
+
+#### Dependencies
+
+- [Robot Operating System (ROS)](https://index.ros.org/doc/ros2/) (middleware for robotics),
+
+
+#### Building
+
+To build from source, clone the latest version from this repository into your  workspace and compile the package using
+
+	cd workspace/src
+	git clone https://github.com/VorteX-co/VAUV/tree/master/Software/vortex_ws/src/vision
+	cd ../
+	rosdep install --from-paths . --ignore-src
+	catkin_make
+
+
+## Usage
+
+
+
+## Config files
+
+
+
+
+## Launch files
+
+
+ 2
+    - **`...`**
+
+* **...**
+
+## Nodes
+#localization
+
+pose_estimation :estimates 3D state of the AUV with the usage of DVl,IMU,point cloud data and visual odometry from the streo camera .
+
+
+#### Subscribed Topics
+fused_date
+
+#### Published Topics
+estimated_pose‏
+#### Services
+
+
+
+#### Parameters
+
+local_SLAM      :generates a series of poses and submaps by scan matching and using a motion filter .
+#### Subscribed Topics
+estimated_pose‏
+
+#### Published Topics
+submaps
+#### Services
+
+
+
+#### Parameters‏
+
+
+
+global_SlAM     : updates the poses/states from the Local SLAM system and then updates the submaps and create a Global 3D map .
```

Comment:
updates the poses/states from the Local SLAM system or using Local SLAM system ? please clarify the description